### PR TITLE
some item model generator mappings

### PR DESCRIPTION
**Pull request by eristhea**
just mapping something that was pointed out by a discord user

### CODE REVIEW

**Comment by eristhea** [comment]
ready for review!

File: mappings/net/minecraft/data/client/ItemModelGenerator.mapping [hunk 1]
```diff
@@ -23,7 +23,11 @@ CLASS net/minecraft/unmapped/C_cfvrpkog net/minecraft/data/client/ItemModelGener
 	METHOD m_llpeiyoc createArmorJson (Lnet/minecraft/unmapped/C_ncpywfca;Ljava/util/Map;Lnet/minecraft/unmapped/C_ncpywfca;)Lcom/google/gson/JsonObject;
 		ARG 1 id
 		ARG 2 textures
+		ARG 3 overrideArmorMaterialId
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/data/client/ItemModelGenerator.mapping]: I see this comes from `TrimModelData#overrideArmorMaterials` which is proposed, but I can't find the source of the proposal.

I think `armorMaterialOverrides` would be a more natural word order and sound less like a boolean or a method name.

File: mappings/net/minecraft/data/client/ItemModelGenerator.mapping [hunk 2]
```diff
@@ -23,7 +23,11 @@ CLASS net/minecraft/unmapped/C_cfvrpkog net/minecraft/data/client/ItemModelGener
...
 	METHOD m_mppeudhx registerArmor (Lnet/minecraft/unmapped/C_vorddnax;Lnet/minecraft/unmapped/C_ncpywfca;Lnet/minecraft/unmapped/C_mgbzsqdv;Lnet/minecraft/unmapped/C_yuycoehb;)V
+		ARG 1 item
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/data/client/ItemModelGenerator.mapping:28]:
```suggestion
		ARG 1 item
		ARG 2 overrideArmorMaterialId
```
It's not the id of the armor

**Review by supersaiyansubtlety — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/data/client/ItemModelGenerator.mapping (hunk 2 above)
Comment (eristhea) [reply mappings/net/minecraft/data/client/ItemModelGenerator.mapping:28]: fixed!

File: mappings/net/minecraft/data/client/ItemModelGenerator.mapping (hunk 1 above)
Comment (eristhea) [reply mappings/net/minecraft/data/client/ItemModelGenerator.mapping]: i agree, i've updated the names in the record

File: mappings/net/minecraft/data/client/ItemModelGenerator.mapping [hunk 3]
```diff
@@ -23,9 +23,10 @@ CLASS net/minecraft/unmapped/C_cfvrpkog net/minecraft/data/client/ItemModelGener
 	METHOD m_llpeiyoc createArmorJson (Lnet/minecraft/unmapped/C_ncpywfca;Ljava/util/Map;Lnet/minecraft/unmapped/C_ncpywfca;)Lcom/google/gson/JsonObject;
 		ARG 1 id
 		ARG 2 textures
-		ARG 3 overrideArmorMaterialId
+		ARG 3 modelId
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/data/client/ItemModelGenerator.mapping]: I don't think this is correct.

It's used here:
```java
jsonObject2.addProperty("model", this.suffixTrim(id, lv.getArmorMaterialOverride(modelId)).toString());
```
where `lv.getArmorMaterialOverride(modelId)).toString()` is passed to `suffixTrim`'s `materialName` param

```suggestion
		ARG 3 materialOverrideId
```

File: mappings/net/minecraft/data/client/ItemModelGenerator.mapping [hunk 4]
```diff
@@ -23,9 +23,10 @@ CLASS net/minecraft/unmapped/C_cfvrpkog net/minecraft/data/client/ItemModelGener
...
 	METHOD m_mppeudhx registerArmor (Lnet/minecraft/unmapped/C_vorddnax;Lnet/minecraft/unmapped/C_ncpywfca;Lnet/minecraft/unmapped/C_mgbzsqdv;Lnet/minecraft/unmapped/C_yuycoehb;)V
 		ARG 1 item
+		ARG 2 modelId
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/data/client/ItemModelGenerator.mapping]: same as above
```suggestion
		ARG 2 materialOverrideId
```

**Review by supersaiyansubtlety — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/data/client/ItemModelGenerator.mapping (hunk 3 above)
Comment (eristhea) [reply mappings/net/minecraft/data/client/ItemModelGenerator.mapping]: ok i get what happened here
i got it right the first time, this is what i meant, but i then proceeded to misunderstand the code when reevaluating the name. embarrassing for me

**Review by supersaiyansubtlety — APPROVED** [review]

**Pull request merged**